app/assets/stylesheets/users.css

```diff
@@ -5,7 +5,7 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
-  font-family: 'Nunito', sans-serif;
+  font-family: 'proxima-nova', sans-serif;
```

Comment:
- [ ] Great job on your progress! You've made valuable updates to the font family in your project by setting it to `font-family: 'proxima-nova', sans-serif;`. However, there's one critical step you might have missed – adding the `proxima-nova` font to your project resources. Unlike web-safe fonts, custom fonts like `proxima-nova` need to be included in your project explicitly. It seems that the font isn't available in your resources, so your project might not be using the desired `proxima-nova` font.
To rectify this, consider adding the `proxima-nova` font to your project. You can find various online resources for `proxima-nova` or use it as a local resource, depending on the licensing terms. Once added, your project should display text in the specified font. If you encounter any issues or have questions about adding custom fonts, feel free to ask for assistance.

Additionally, if you want to explore how to efficiently include custom fonts in your Rails project, you can refer to resources like [this link](https://world.hey.com/aesmail/custom-fonts-in-production-in-rails-7-0-d3765485).
Font link to download: [Proxima Nova](https://befonts.com/proxima-nova-font.html) 
Keep up the good work! 🚀🔤